scripts/packages/chocolatey/bazel.nuspec.template

```diff
@@ -75,8 +75,9 @@ Supply like `--params="/option:value ..."` ([see docs for --params](https://gith
       <dependency id="chocolatey-uninstall.extension" />
     </dependencies>-->
     <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0.7"/>
       <dependency id="jdk8" version="[8.0.102,)"/>
-      <dependency id="msys2" version="[20160719.1,)"/>
+      <dependency id="msys2" version="[20160719.1.0]"/>
```

Comment:
No, please don't. As i understand it, both 20160719.1.0 and 20160719.1.1 are good, but versions with a more recent (i.e. where the first part of the version string is newer, e.g. 20170130.x.y) seem to be bad. So the pin should be [20160719.1.0,20160719.1.1].